Marksman/Marksman.fsproj

```diff
@@ -1,39 +1,34 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-    <PropertyGroup>
-        <OutputType>Exe</OutputType>
-        <TargetFramework>net6.0</TargetFramework>
-        <AssemblyName>marksman</AssemblyName>
-    </PropertyGroup>
-
-    <ItemGroup>
-        <Compile Include="Misc.fs" />
-        <Compile Include="Text.fs" />
-        <Compile Include="Cst.fs" />
-        <Compile Include="Parser.fs" />
-        <Compile Include="Index.fs" />
-        <Compile Include="Workspace.fs" />
-        <Compile Include="Compl.fs" />
-        <Compile Include="Semato.fs" />
-        <Compile Include="Diag.fs" />
-        <Compile Include="State.fs" />
-        <Compile Include="Server.fs" />
-        <Compile Include="Program.fs" />
-    </ItemGroup>
-
-    <ItemGroup>
-        <PackageReference Update="FSharp.Core" Version="6.0.4" />
-        <PackageReference Include="FSharp.SystemCommandLine" Version="0.10.0-beta" />
-        <PackageReference Include="FSharpPlus" Version="1.2.3" />
-        <PackageReference Include="Markdig" Version="0.30.2" />
-        <!--        <PackageReference Include="Ionide.LanguageServerProtocol" Version="0.3.1" />-->
-        <PackageReference Include="Serilog" Version="2.11.0" />
-        <PackageReference Include="Serilog.Sinks.Console" Version="4.0.1" />
-    </ItemGroup>
-
-    <ItemGroup>
-        <ProjectReference Include="..\LanguageServerProtocol\LanguageServerProtocol.fsproj" />
-        <ProjectReference Include="..\MarkdigPatches\MarkdigPatches.csproj" />
-    </ItemGroup>
-
-</Project>
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is it some tool that reformat this file?

Reply:
:shrug: no idea

_could be_ line endings? Are developing this on a Windows?

Reply:
I don't think it's line endings, bc the indentation here is 2 spaces vs 4 before. Could you rebase on latest master and run `make fmt` -- it should reformat the proj file.